Makefile

```diff
@@ -0,0 +1,18 @@
+# Docker is the only dependency here
+# Build the dashboard image for host or cross-compile
+# This Makefile can only be run on a amd64 host
+
+build-image:
```

Comment:
Sounds good to me. This is just the first way I could think of to do this. I'm open to all improvements and ideas, and having a `gulp-in-docker.sh` script is a better idea than having a `Makefile`.

I think it's best to split this PR in some logical pieces. This one could focus on the `Dockerfile` first, and how we should pass the arguments etc.